packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -193,7 +193,7 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
         </Panel>
         {!isLoadingPeerProfile && isValidURL && (
           <Panel header={t('peerList.panels.information')}>
-            <ChooseEditImage disabled image={profile?.logo} />
+            {(profile?.logo )?(<ChooseEditImage disabled image={profile?.logo} />):(<ControlLabel>{t('chooseEditImage.header')}<Placeholder.Graph/> </ControlLabel>)}
```

Comment:
I like your solution but i think it should be inside the `ChooseEditImage` component. Since it have the `disabled` flag you can show the placeholder whenever `disabled==true` and `image` being undefined.